pkg/generated/openapi/BUILD

```diff
@@ -17,6 +17,7 @@ openapi_library(
         "pkg/kubelet/apis/kubeletconfig/v1beta1",
         "pkg/proxy/apis/kubeproxyconfig/v1alpha1",
         "pkg/version",
+        "vendor/github.com/kubernetes/repo-infra/kazel",
```

Comment:
welp, the openapi generator in kazel is picking up itself, erroneously. I'll fix this.

Reply:
thanks